mmdet/datasets/lvis.py

```diff
@@ -277,17 +278,15 @@ def load_annotations(self, ann_file):
 
         try:
             import lvis
-            assert lvis.__version__ >= '10.5.3'
+            if lvis.__version__ >= '10.5.3':
```

Comment:
This will trigger an AttributeError for official LVIS

Reply:
Thx! Fixed in #4958